jmxtrans-output/jmxtrans-output-kafka/src/test/resources/kafka.properties

```diff
@@ -0,0 +1,128 @@
+#
```

Comment:
Didn't you say that config files are Apache licensed? Though to be honest, I'm not sure there is enough creation in this to be copyrightable...